.github/workflows/deployment.yml

```diff
@@ -34,4 +48,4 @@ jobs:
           # Application name on the CapRover server. Must exists.
           appname: 'iatipublisher-staging'
           # Branch which will be deployed
-          branch: main
\ No newline at end of file
+          branch: develop
```

Comment:
rename `develop` branch to `staging` branch

Reply:
resolve: rename `develop` branch to `staging` branch